FND/DatesAndTimes/BusinessDates.rdf

```diff
@@ -135,26 +136,18 @@ They are modularized this way to minimize the ontological committments that are
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business recurrence interval</rdfs:label>
-		<skos:definition>a recurrence interval that is specified using a business recurrence interval convention</skos:definition>
+		<skos:definition>recurrence interval that is specified using a business recurrence interval convention</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-dt-bd;BusinessRecurrenceIntervalConvention">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-bd;Convention"/>
 		<rdfs:label>business recurrence interval convention</rdfs:label>
-		<skos:definition>a convention for modeling recurring days, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
+		<skos:definition>convention that specifies how recurring days should be handled, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
```

Comment:
I don't see why t-bill auction is here, and it's very jurisdiction-specific

Reply:
It was there before - all I did was fix the definition.  And there are t-bills referenced in the FpML that may not be limited to the US, including Australian t-bills ... although those could be Australian dollar but US t-bills.